sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2025-2026.bpmn

```diff
@@ -853,7 +853,7 @@ dmnX (comes from dmn)</bpmn:text>
     <bpmn:intermediateThrowEvent id="Event_1yndxhv" name="awardEligibilityCSGD">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=if (calculatedDataTotalAssessedNeed &#62;= 1)&#10;    and (calculatedDataTotalEligibleDependants &#62;= 1)&#10;    and (calculatedDataTotalFamilyIncome &#60; dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDThresholdIncome)&#10;then true&#10;else false" target="awardEligibilityCSGD" />
+          <zeebe:output source="=if (calculatedDataTotalAssessedNeed &#62;= 1)&#10;    and (calculatedDataTotalEligibleDependants &#62;= 1)&#10;  and (calculatedDataDependants11YearsOrUnder &#62;= 1 or calculatedDataDependants12YearsOverOnTaxes &#62;= 1)&#10;    and (calculatedDataTotalFamilyIncome &#60; dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDThresholdIncome)&#10;then true&#10;else false" target="awardEligibilityCSGD" />
```

Comment:
Even though that does not cause any issue, the eligibility seems to have extra criteria than the AC.
![image](https://github.com/user-attachments/assets/3cc54612-bc17-4a58-8c17-2d80471a09cd)